jira.py

```diff
@@ -1,8 +1,14 @@
 import os
 import sys
 
-from lib.jira_conn import JiraAPIClient
+import pandas as pd
 
+from lib.jira_conn import JiraAPIClient
+from database import (
+    Database,
+    # ReportJiraQARequests
+)
+from utils.datetime_utils import DatetimeUtils as dt
 
 # JQL query All QA Requests since 2022 filter_id: 13856
 FILTER_ID_ALL_REQUESTS_2022 = "13856"
```

Comment:
OK for now, but should prob move to [constants.py](https://github.com/mozilla-mobile/test-dashboard/blob/master/utils/constants.py)

Reply:
yeah, that makes sense. I can make the changes in this PR